src/activities/content/ContentEntity.js

```diff
@@ -3,7 +3,7 @@ import { Rels } from '../../hypermedia-constants';
 import ContentHelperFunctions from './ContentHelperFunctions.js';
 
 export const CONTENT_TYPES = {
-	htmlfile: 'htmlfile',
+	file: 'content-file',
```

Comment:
just a general observation: I've noticed sometimes constants are referred to as `file` but sometimes `contentFile`. is there any distinction here, and should we be consistent? for example in this same file:
```
getFileHref() {
		return ContentHelperFunctions.getHrefFromRel(Rels.Content.fileEntity, this._entity);
	}
```
should this be `getContentFileHref()`?

Reply:
I see what you mean. I'll do a quick scan and see if there are any places where it should be `contentFile`.